camel-core/pom.xml

```diff
@@ -115,6 +115,7 @@
     </camel.osgi.activator>
     <!-- do not skip any tests by default -->
     <platform.skip.tests/>
+    <camel.automatic.module.name>org.apache.camel</camel.automatic.module.name>
```

Comment:
Should it not be org.apache.camel.core

Reply:
Oh reverse internet domain name then its: core.camel.apache.org

Can we double check this to be sure?

Reply:
Hi @davsclaus the camel website is currently at http://camel.apache.org/ so our reverse internet domain is 'org.apache.camel' which also happens to be our principal exported API package. Hence I propose 'org.apache.camel' as the automatic module name but 'org.apache.camel.core' is fine with me. Here are some other projects for comparison:

[Mapstruct](https://github.com/mapstruct/mapstruct/issues/1224) -> org.mapstruct
[Mockito](https://github.com/mockito/mockito/issues/1189) -> org.mockito
[RxJava](https://github.com/ReactiveX/RxJava/issues/5618) -> io.reactivex.rxjava2

For our Camel components, dataformats and spring starters we can set the automatic module name to the primary package. We can also set the name to 'org.apache.camel.X' for everything under components/X and 'org.apache.camel.springboot.X' for all our spring starters to shorten the names a little and for consistency. I have a preference for the second option.

Let me know what you think,

John.

Reply:
Thanks. I suggest we wait with this until 2.21 release so we can debate in the community what the naming should be.

Reply:
I logged a ticket: https://issues.apache.org/jira/browse/CAMEL-11887